### ISSUE TITLE
Request for new or update to existing KB article : Lovell-editor specific instructions

### ISSUE THREAD

**Issue by dsinla** [department-of-veterans-affairs/va.gov-cms#11687]
## Background context
**What is the request for?**
Lovell editors will create content like any other VAMC editor, with a few special cases that need to be documented. 
Given that the scope of Lovell is small (just 4 editors). we should not edit existing KB articles, but rather create new Lovell-specific articles.

I'm suggesting:

- Authoring content for Lovell
	Overview content and switcher description

- Creating content for Lovell 
	Assigning the proper section 

- Changing the assigned section for Lovell content 
	Checking for existence of content
	Assigning proper section 

[Draft of content ](https://docs.google.com/document/d/1wkeJNJxen_TNYnGqlzDzjFCSKq2vRxt_6dy6pkuspR0/edit#)

**Person and team of requester:** 
- David Shannon, Facilities team 

**Priority or deadline for request**

  Choose either Deadline or a priority:
  
- [x] Deadline: KBs need to be in place prior to training - targeting Sprint 73, so Ideally KBs are completed in Sprint 72
- [ ] 
- [ ] Priority:
  - [ ] Critical - Without this, there is a large negative impact on the majority of editors causing negative consequences within the CMS; Should be completed as soon as possible
  - [ ] High - Without this, there is negative impact on some editors causing negative consequences within the CMS; should be completed within the current sprint
  - [ ] Medium - Without this, there is confusion for editors but not causing negative consequences within the CMS; Should be completed within the next 1-2 sprints
  - [ ] Low - This is not currently causing confusion for many editors but would be a nice to have or is something that may be more impactful in the future; Should be completed within the next 3-4 sprints
  

**Is the article associated with an ongoing product or feature update?**
- [x] Yes #9655
- [ ] No

  
## If requesting a new article:

**Audience:**
Who is this intended for? Who would need this information? 
VAMC Editors of Lovell 

**Related KB articles, if applicable:** 
- [If you know that related KB articles already exist, link them here]

  
**Any other relevant documentation, if applicable:** 
- [for example, could be the VA content style guide]

## Process
1. Sitewide CMS and Helpdesk teams will review request and confirm no current, applicable KB article/content exists.
2. Refine request and prioritize
3. Update KB article tracker to indicate that the article has been requested and is in progress
4. Assign to appropriate product team
5. Draft article content 
6. Sitewide CMS team will review for formatting and writing style guidelines
7. Helpdesk team will review for functionality and editorial tone
8. Publish
9. Update KB article tracker to indicate that the article has been published

## Acceptance Criteria
- [x] 3 Articles have been created using [KB content style guidelines](https://va-gov.atlassian.net/wiki/spaces/VAGOV/pages/1771831299/Knowledge+Base+content+style) 
- [x] ~~3~~ [2 Articles](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/11687#issuecomment-1470578119) have been created at a time preceding full feature complete, and now they are likely out of date, review current written articles and add additional information needed to reflect the actual functionality, remove any now incorrect information--
- https://prod.cms.va.gov/lovell-federal-va-health-care/vamc/creating-content-for-lovell-federal-health-care
- https://prod.cms.va.gov/help/vamc/assigning-lovell-federal-health-care-content-to-a-section
- ~~https://prod.cms.va.gov/lovell-federal-va-health-care/vamc/assigning-sections-for-lovell-federal-health-care-content~~
- ~~https://prod.cms.va.gov/lovell-federal-va-health-care/vamc/changing-the-assigned-section-for-lovell-federal-health-care-content~~
- [x] I expect that information surrounding events and (news releases + stories) are addressed within the scope of the knowledge base articles for Lovell, this may be addressed by creating new articles OR by incorporating this information into the existing articles that have been drafted
- [x] Article is reviewed by Sitewide CMS
- [x] Article is reviewed by Helpdesk
- [x] Article published to KB 
- [x] Impacts on other KB articles have been considered and any necessary changes have been made
- [x] Once published, the person requesting the article has been notified

**Comment by dsinla** [comment]
First draft of topics:

https://prod.cms.va.gov/lovell-federal-va-health-care/vamc/creating-content-for-lovell-federal-health-care
https://prod.cms.va.gov/lovell-federal-va-health-care/vamc/assigning-sections-for-lovell-federal-health-care-content 
https://prod.cms.va.gov/lovell-federal-va-health-care/vamc/changing-the-assigned-section-for-lovell-federal-health-care-care

**Comment by JQuaggles** [comment]
@omahane / @JayDarnell / @davidmpickett 
Can folks review these three drafts that are in prod kb and leave comments here?

**Comment by JayDarnell** [comment]
I've added this to my to-do list and will take a look as soon as possible.

**Comment by davidmpickett** [comment]
The checklist says Sitewide CMS and Helpdesk need to review. Have we identified who will handle those reviews? That seems like the blocker here. Per [this comment](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/10027) it sounds like these drafts already got Facilities Dev review.

**Comment by davidmpickett** [comment]
Feedback:

- These articles use the word "page" in instances that apply to non-page content. 
- Under "Dual publishing" it might be helpful is an exhaustive list of the content types that do or do not support Dual Publishing. That is one of the most confusing aspects to this whole mess.

**Content Types that support Dual Publishing:**
- Events
- News Release
- Story
- VAMC Detail Page

**Content Types that don't (or shouldn't) support Dual Publishing:**
- VAMC System Health Service (pending #11980)
- VAMC Facility Health Service (pending #11980)
- VAMC Facility Non-clinical Service (pending #11980)
- VAMC facilities
- VAMC Systems
- VAMC System Register for Care
- Health Services List
- Events List 
- News Releases List
- Stories List
- VAMC System Billing and Insurance
- VAMC System Locations List
- VAMC System Medical Records Office
- VAMC System Operating Status
- VAMC System Policies Page
- VAMC System Register for Care

**Content types I'm not sure about**
- VAMC System Banner Alert with Situation Updates
- Staff Profile?

**Comment by davidmpickett** [comment]
### Feedback on "Assigning sections for Lovell Federal health care content"

Under "To create a page with content for only VA or for only TRICARE"

> If there is a corresponding page, the toggle will appear at the top of the published page. 

This doesn't seem to adequately explain the requirements for having two non-dual published pages have a toggle

**Comment by omahane** [comment]
I agree with @davidmpickett about clarifying the dual publishing description, so I added a table with the content types and which support dual publishing or not.

I also added a "Placeholder" stamp on the images with the URLs from Tugboat, as these will need to be replaced with images from the production environment.

**Comment by Dottisea** [comment]
Is this still blocked? Both of the tickets that I can see that are defined as blockers ([Lovell Header naming convention change](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12092) and #11980) - I would like to move this to ready for refinement for our next refinement meeting...

**Comment by Dottisea** [comment]
Confirm what the articles are that need review/editing.

**Comment by omahane** [comment]
I'm keeping track of the review process of these KBAs here: https://docs.google.com/spreadsheets/d/1INayZyTeRVba31f9yDkZIJr9HoBKPC5F_FwetK2Po_8/edit#gid=0

**Comment by omahane** [comment]
After talking with @laflannery, we determined to combine the "section assigning/changing" articles into one, leaving only
https://prod.cms.va.gov/help/vamc/creating-content-for-lovell-federal-health-care
https://prod.cms.va.gov/help/vamc/assigning-lovell-federal-health-care-content-to-a-section
as the remaining articles specific to Lovell.
